app/cafe.py

```diff
@@ -0,0 +1,22 @@
+import datetime
+from app.errors import NotVaccinatedError,\
+    NotWearingMaskError,\
+    OutdatedVaccineError
+
+
+class Cafe:
+    def __init__(self, name: str):
+        self.name = name
+        self.today = datetime.date.today()
+
+    def visit_cafe(self, visitor: dict):
+        if "vaccine" in visitor.keys():
+
+            if visitor["vaccine"]["expiration_date"] < self.today:
+                raise OutdatedVaccineError("OutdatedVaccineError")
```

Comment:
"OutdatedVaccineError" is not a descriptive message. You should add some information about an error in the corresponding message.